lib/sisimai/bite/email/sendmail.rb

```diff
@@ -0,0 +1,290 @@
+module Sisimai::Bite::Email
+  module Sendmail
+    # Sisimai::Bite::Email::Sendmail parses a bounce email which created by
+    # v8 Sendmail. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Sendmail.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->